cmd/kubeadm/app/cmd/phases/reset/cleanupnode.go

```diff
@@ -128,7 +128,7 @@ func resetConfigDir(configPathDir, pkiPathDir string) {
 	fmt.Printf("[reset] Deleting contents of config directories: %v\n", dirsToClean)
 	for _, dir := range dirsToClean {
 		if err := CleanDir(dir); err != nil {
-			klog.Warningf("[reset] Failed to remove directory: %q [%v]\n", dir, err)
+			fmt.Printf("[reset] WARNING: Failed to delete contents of %q directory: [%v]\n", dir, err)
```

Comment:
Reasons for using `fmt.Printf`  kubernetes/kubeadm#1913

Reply:
Should it be `klog.V(1).Warningf`? [1]

Reply:
Not quite sure! But does `klog` support `klog.V(1).Warningf`? 
I didn't find that usage from the code base (only `klog.Warningf` could be found).
I believe the necessary warnings should be displayed by default. WDYT?

Reply:
In klog, log levels exist and are applicable only to the "info" severity. :)

Reply:
the discussion in kubernetes/kubeadm#1913
is not clear yet. if users don't want klog output they can always pipe it to /dev/null as its printed to stderr.

let's keep the klog.Warningf for now and change all of those in a single PR once we decide after the discussion in kubernetes/kubeadm#1913

other than this LGTM